ROADMAP.md

```diff
@@ -58,23 +58,43 @@ Please file an issue to make suggestions on this roadmap!
 
 ## v0.9.8
 
+  * Kubernetes 1.7
+  * More and [more RBAC support](https://github.com/kubernetes-incubator/kube-aws/pull/675#issuecomment-303660360) (@camilb, @c-knowles)
   * Experimental support for kube-aws plugins
   * Tiller installed by default
     * For use from the plugin support
+  * Cluster-provisioning observability improvements
+    * Streaming stack events & journald logs (@jollinshead)
 
 ## v0.9.9
 
+  * Kubernetes 1.8
   * RBAC enabled by default
+  * Security improvements
+    * NodeRestriction admission controller + Node authorizer + Kubelet’s credential rotation (@danielfm)
+  * Scalability improvements
+    * More efficient node draining(@danielfm)
   * [Optional] Several kube-aws core features as plugins
 
 ## v0.9.10
 
-  * kubeadm support to simplify k8s components configuration
-    * while reducing the amount of code required in kube-aws
-  
-## v0.9.x
+  * kubeadm support to simplify k8s components configuration (#654)
+    * Reduces the amount of code required in kube-aws
+    * To better follow upstream improvements on how k8s components are deployed
+  * [More manageable Calico + Flannel](https://github.com/kubernetes-incubator/kube-aws/pull/675#issuecomment-303669142) (@redbaron)
+  * istio integration
```

Comment:
If the upstream improvement made it in time, this can be re-targeted to kube-aws v0.9.9 which targets k8s 1.8